presto-orc/src/test/java/com/facebook/presto/orc/TestOrcRecordReader.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+import com.facebook.presto.hadoop.shaded.com.google.common.collect.ImmutableList;
+import com.facebook.presto.hadoop.shaded.com.google.common.collect.ImmutableMap;
+import com.facebook.presto.orc.metadata.OrcType;
+import com.facebook.presto.spi.type.BigintType;
+import com.facebook.presto.spi.type.BooleanType;
+import com.facebook.presto.spi.type.DateType;
+import com.facebook.presto.spi.type.DoubleType;
+import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.spi.type.VarcharType;
+import com.facebook.presto.type.ArrayType;
+import com.facebook.presto.type.MapType;
+import com.facebook.presto.type.RowType;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertThrows;
+
+@Test
+public class TestOrcRecordReader
+{
+    @Test
+    public void testValidateOrcType()
+    {
+        final List<OrcType> types = ImmutableList.of(
+                // 0
+                new OrcType(OrcType.OrcTypeKind.STRUCT,
```

Comment:
When you put arguments on multiple lines, always put the argument on its own line.